typescript/infra/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@abacus-network/core": "^0.2.1-beta3",
     "@abacus-network/deploy": "^0.2.1-beta3",
+    "@abacus-network/helloworld": "0.2.1-alpha",
```

Comment:
See comment in your draft PR, would be good for this to match the SDK version it's using